app/controllers/photos_controller.rb

```diff
@@ -3,5 +3,20 @@ class PhotosController < ApplicationController
 
   def index
     @user = current_user
+    @photos = flickr_user.photos if user_has_flicker_service?
+  end
+
+  private
+
+  def flickr
+    @flickr ||= Flickr.new(ENV['FLICKR_KEY'])
+  end
+
+  def flickr_user
+    @flickr_user ||= flickr.users(@user.flickr_username)
+  end
+
+  def user_has_flicker_service?
```

Comment:
Looks like there is typo here
```suggestion
  def user_has_flickr_service?
```